packages/govern-console/src/components/Wallet/Wallet.tsx

```diff
@@ -115,72 +80,76 @@ const Wallet = ({}) => {
     }
   }, [account]);
 
+  const getDisplayText = (displayText: string, isAddress: boolean) => {
+    if (isAddress) {
+      const formattedAddress =
+        displayText.slice(0, 5) +
+        '...' +
+        displayText.slice(displayText.length - 5, displayText.length - 1);
+      return formattedAddress;
```

Comment:
there's a util function to shorten address here..
https://github.com/aragon/govern/blob/develop/packages/govern-console/src/utils/account.tsx

Reply:
looks like `getDisplayText` is only used in 1 spot and the `isAddress` is always true, may be we can remove this function and just use the `getTruncatedAccountAddress()` function.